src/styles/index.scss

```diff
@@ -1,3 +1,172 @@
-body {
+$main-accent: #060b35;
```

Comment:
it's a good practice to create separated files for logic blocks and import them to index.scss

So, please create such files and import using order below:
1) variables.scss (for variables)
2) common.scss (for common styles such as `*`, `body`, `html`  selectors
3) stars.scss (for stars styles)
4) card.scss (for card.scss)